src/framework/global/hash.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2024 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MUSE_GLOBAL_HASH_H
+#define MUSE_GLOBAL_HASH_H
+
+#include "async/notification.h"
+#include "async/asyncable.h"
+#include <functional>
+#include <tuple>
+
+namespace muse {
+size_t hash(int a, int b, char c)
```

Comment:
Hm... in practice this isn't as beautiful as I had imagined. I had in mind to create a general hash combine function that could work with any collection of hashable types, but perhaps you're right that it's a bit overkill to do that just to support a single case. 

But now that it is such a specific method for a specific case, it makes more sense to put it as a static method near the place where it is used. 

(Sorry for asking another revision about this; it's a case of "advancing insight" as we'd call that in Dutch 🙂)